### PR TITLE
Start preparing for v238-rc1

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -2,13 +2,235 @@ systemd System and Service Manager
...
-          and they will take precedence. If the host's root mount point is used,
-          there is no change in behaviour.
+
+          Users can always specify BindPaths= or InaccessiblePaths= as
+          overrides, and they will take precedence. If the host's root mount
+          point is used, there is no change in behaviour.
+
+        * New bind mounts and file system image mounts may be injected into the
+          mount namespace of a service (without restarting it). This is exposed
+          as 'systemctl mount-image <unit> <image>…'.
```
Comment (bluca) [inline NEWS:83]: should this also show 'systemctl bind' for the bind mounts case, since it's mentioned?

File: NEWS [hunk 2]
```diff
@@ -2,13 +2,235 @@ systemd System and Service Manager
...
+        * The hardware database has been extended with a list of fingerprint
+          readers that correctly support autosuspend using data from libfprint.
+
+        * systemd-resolved can now answer DNSSEC questions through the stub
+          resolver interface in a way that allows local clients to do DNSSEC
+          validation themselves. For a question with DO+CD set, it'll proxy the
+          DNS query and respond with a mostly unmodified packet received from
+          the upstream server.
+
+        * systemd-nspawn gained a new -ambient-capability= setting
```
Comment (bluca) [inline NEWS:160]: s/-ambient/--ambient/

File: NEWS (hunk 1 above)
Comment (keszybz) [reply NEWS:83]: Please just submit a follow-up pr... This will be quicker than handling each typo and omission through comments.

Comment (bluca) [reply NEWS:83]: sure, will do once this is merged

**Review by bluca — APPROVED** [review]
NEWS update and python script change look good to me - the rest is autogenerated so haven't checked. LGTM

**Pull request merged**